server/bin/pbench-index.py

```diff
@@ -44,38 +44,8 @@ def main(options, name):
        All exceptions are caught and logged to syslog with the stacktrace of
        the exception in a sub-object of the logged JSON document.
 
-       Status codes used by es_index and the error below are defined from the
-       list below to maintain compatibility with the previous code base when
-       pbench-index was a bash script and invoked index-pbench (yes, a bit
-       confusing), the predecessor to this program.  The codes were used to
-       sort the errors we encountered processing tar balls in to categories
-       of retry or not:
-
-            0 - normal, successful exit, no errors
-            1 - Operational error while indexing
-            2 - Configuration file not specified
-            3 - Bad configuration file
-            4 - Tar ball does not contain a metadata.log file
-            5 - Bad start run date value encountered
-            6 - File Not Found error
-            7 - Bad metadata.log file encountered
-            8 - Error reading a mapping file for Elasticsearch templates
-            9 - Error creating one of the Elasticsearch templates
-           10 - Bad hostname in a sosreport
-           11 - Failure unpacking the tar ball
-           12 - generic error, needs to be investigated and can be retried
-                after any indexing bugs are fixed.
-
-       Return Values (now a sub-set of the original status codes above):
-         0 - Successfully processed all tar balls (errors processing tar
-             balls are reported in the logs and in index status reports)
-         1 - Failed to process one or more tar balls for unknown reasons
-             (see logs)
-         2 - Missing configuration file
-         3 - Invalid configuration file
-         8 - Unable to load and process expected mapping files
-         9 - Unable to update index templates in configured Elasticsearch
-             instance
+       Status codes used by the error below are defined from the list present
+       in indexing-tarballs.
```

Comment:
I'm not sure this is of much use anymore. One has to read the code to see that it's referencing the `error_code` attribute of the created `Index` object, and we don't (and probably don't want to) provide any more details in this context.

We need to either expand this to be useful _without_ reading the code, or we might as well delete it. Since code importing a module (or creating an object) usually isn't responsible for documenting the details of that imported code, I'm inclined to just remove this.

Reply:
I think this is my fault.  My earlier suggestion was to remove the commend but that leaving a "reference to where those definitions can be found might be good".  I repent...let's just remove it.  (Sorry for the bad lead, Riya!)

Reply:
it was a good suggestion but maybe didn't work here.